.gitignore

```diff
@@ -21,6 +21,9 @@ dist/
 .python-version
 __pycache__
 
+# this is temporary
+test.py
+
```

Comment:
this should be removed or [locally ignored](https://git-scm.com/docs/gitignore#_configuration) (global `.gitignore` or local `.git/info/exclude`)